layers/meta-tegra-support/recipes-test/tegra-tests/vpi1-tests_1.0.bb

```diff
@@ -0,0 +1,25 @@
+DESCRIPTION = "Scripts for testing Tegra VPI samples"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COREBASE}/meta/COPYING.MIT;md5=3da9cfbcb788c80a0384361b4de20420"
+
+SRC_URI = "\
+    file://run-vpi1-tests.sh.in \
+"
+
+COMPATIBLE_MACHINE = "(tegra)"
+
+S = "${WORKDIR}"
+B = "${WORKDIR}/build"
+
+do_compile() {
+    sed -e's,@SOC_FAMILY@,${SOC_FAMILY},g' \
+        ${S}/run-vpi1-tests.sh.in >${B}/run-vpi1-tests.sh
```

Comment:
If you detect the SoC type at runtime, you don't need this.

Reply:
Agree